test/runtests.jl

```diff
@@ -19,30 +19,59 @@ img_eps = isinteractive() ? 1e-2 : 10e-2
 #     image_comparison_facts(:gadfly, skip=[4,6,23,24,27], eps=img_eps)
 # end
 
+facts("GR") do
+    @fact gr() --> Plots.GRBackend()
+    @fact backend() --> Plots.GRBackend()
+
+    image_comparison_facts(:gr, eps=img_eps)
+end
+
 facts("PyPlot") do
     @fact pyplot() --> Plots.PyPlotBackend()
     @fact backend() --> Plots.PyPlotBackend()
 
-    image_comparison_facts(:pyplot, skip=[6,25,30,31], eps=img_eps)
+    image_comparison_facts(:pyplot, skip=[2,31], eps=img_eps)
```

Comment:
That's... slightly confusing :-) But thanks for noticing, you really have a sharp eye here.
Maybe it's good to do that, though we're only testing a bit of the functionality in StatPlots, we're at least testing the functionality that's in the juliaplots documentation.

What do you say, include this?
If we do, we need to remove the 'using Plots' statements though. StatPlots reexports Plots so this will/may cause issues with function versions.